mkdoc.zsh

```diff
@@ -166,6 +174,8 @@ events=$(<<EOF
 	//                      you may get hundreds of Write events, so you
 	//                      probably want to wait until you've stopped receiving
 	//                      them (see the dedup example in cmd/fsnotify).
+	//                      Some systems may send Write event for directories
+	//                      when the directory content changes.
```

Comment:
This behaviour is kind of annoying, IMO; if you create a new file or directory it sends a Write on the directory, too:

	>  fsnotify watch ./test/...

	> md NEWDIR
	19:36:19.7176   1 CREATE        "test\\NEWDIR"

	> cd NEWDIR
	> md SUBDIR
	19:36:29.4840   2 CREATE        "test\\NEWDIR\\SUBDIR"
	19:36:30.4888   3 WRITE         "test\\NEWDIR"

	> cd SUBDIR
	> echo x >newfile
	19:36:56.9763   4 CREATE        "test\\NEWDIR\\SUBDIR\\newfile"
	19:36:56.9786   5 WRITE         "test\\NEWDIR\\SUBDIR\\newfile"
	19:36:58.2154   6 WRITE         "test\\NEWDIR\\SUBDIR"

	> cd ../..
	> echo asd >zxczxc
	19:38:40.8707   7 CREATE        "test\\zxczxc"
	19:38:40.8741   8 WRITE         "test\\zxczxc"

This is also the case with non-recursirve watches:

	> fsnotify watch ./test2

	> md asd
	19:45:37.6129   1 CREATE        "test2\\asd"

	> echo zxc > zxczxc
	19:45:42.9015   2 CREATE        "test2\\zxczxc"
	19:45:42.9045   3 WRITE         "test2\\zxczxc"

	> cd asd
	> echo zxc > zxczxc
	19:45:49.0316   4 WRITE         "test2\\asd"

It's also inconsistent: you don't get this Write event on the root directory.

The reason is because it seems to use timestamps:

> FILE_NOTIFY_CHANGE_LAST_WRITE
>
> Any change to the last write-time of files in the watched directory or subtree
> causes a change notification wait operation to return. The operating system
> detects a change to the last write-time only when the file is written to the
> disk. For operating systems that use extensive caching, detection occurs only
> when the cache is sufficiently flushed. 
>
> FILE_ACTION_MODIFIED
> 
> The file was modified. This can be a change in the time stamp or attributes. 

I don't see any way to change or disable this. So ... it is what it is, I guess :-/